src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaComparePublishProjectChangesOperation.cs

```diff
@@ -0,0 +1,79 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlServer.Dac.Compare;
+using Microsoft.SqlTools.ServiceLayer.SchemaCompare.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+using Microsoft.SqlTools.Utility;
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+
+namespace Microsoft.SqlTools.ServiceLayer.SchemaCompare
+{
+    /// <summary>
+    /// Class to represent an in-progress schema compare publish project changes operation
+    /// </summary>
+    class SchemaComparePublishProjectChangesOperation: ITaskOperation
```

Comment:
The code in this class is pretty similar to the one for SchemaComparePublishDatabaseChangesOperation. Can this be refactored so they both inherit from the same base class and can reuse some code?

Reply:
Oh yes, will look into how to implement this!